cmd/kwil-admin/nodecfg/toml.go

```diff
@@ -115,7 +115,9 @@ private_key_path = "{{ .AppConfig.PrivateKeyPath }}"
 # TCP address for the KWILD App's JSON-RPC server to listen on
 jsonrpc_listen_addr = "{{ .AppConfig.JSONRPCListenAddress }}"
 
-# Unix socket or TCP address for the KWILD App's Admin GRPC server to listen on
+# Unix socket or TCP address for the KWILD App's Admin GRPC server to listen on.
```

Comment:
Admin service does not use GRPC anymore, so it might be worth updating this comment